lib/helper.rb

```diff
@@ -0,0 +1,28 @@
+require 'json'
+
+module Helper
+
```

Comment:
Please fix hound errors once you've submitted your result - it makes it quite hard to review your code otherwise!